### PR TITLE
[FAT- 16038] - Extend Central Ordering Karate tests coverage

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://folio-org.atlassian.net/browse/FAT-16038
## Approach
- Do unopen the order, check appropriate state of Inventory 
-  Inherit Change instance connection test when order has related items from different 
- Change affiliation in Piece and check that item and holding was re-created in correct 
- Open 1 order with Create Instance, Hodling, Item, and another with Create Instance, Holding. In both cases pieces should have receivingTenantId populated

### CODE REVIEW

**Comment by azizbekxm** [comment]
image

File: acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature [hunk 1]
```diff
@@ -465,23 +465,22 @@ Feature: Open order with member tenant location and verify instance, holding, an
     And match physicalItemAfterOpenOrder2 != null
     And match physicalItemAfterOpenOrder2.status.name == 'Order closed'
 
-
-  Scenario: Verify existance of Inventory in member tenant after reopining order with deleteHoldings=true
```
Comment (azizbekxm) [inline acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature:469]: Seems diff showing confusing defference for this feature. Actually these scenarios duplicate and exissts in reopen-and-change-isntance connection feature. Removed duplicate features and added new test cases.
https://github.com/folio-org/folio-integration-tests/blob/7f79cbc6777b9736a72672f54164e31a48284cf8/acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature#L469

https://github.com/folio-org/folio-integration-tests/blob/7f79cbc6777b9736a72672f54164e31a48284cf8/acquisitions/src/main/resources/thunderjet/consortia/features/reopen-and-change-instance-connection-order-with-locations-from-different-tenants.feature#L34

File: mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature [hunk 2]
```diff
@@ -75,61 +75,89 @@ Feature: mod-consortia integration tests
...
-
-  Scenario: Sharing Instances api tests
-    * call read('features/sharing-instance.feature')
-
-  Scenario: Sharing Settings api tests
-    * call read('features/sharing-setting.feature')
-
-  Scenario: Sharing Patron Groups Settings api tests
-    * call read('features/sharing-patron-groups-setting.feature')
+#  Scenario: Create ['central', 'university', 'college'] tenants and set up admins
```
Comment (SerhiiNosko) [inline mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature:78]: some commented code to revert

File: mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature [hunk 3]
```diff
@@ -75,61 +75,89 @@ Feature: mod-consortia integration tests
...
+#    * call read('features/sharing-setting.feature')
+#
+#  Scenario: Sharing Patron Groups Settings api tests
+#    * call read('features/sharing-patron-groups-setting.feature')
 
   Scenario: Destroy created ['central', 'university', 'college'] tenants
-    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: '#(universityTenant)'}
-    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: '#(collegeTenant)'}
-    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: '#(centralTenant)'}
+    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: 'central1719409122040'}
```
Comment (BKadirkhodjaev) [inline mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature:133]: Can this be stuffed into some array and for-looped?

Comment (BKadirkhodjaev) [reply mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature:133]: Or is this a temporary cleanup that got accidentally committed?

Comment (azizbekxm) [reply mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature]: Didn't expect this changes, I think it come from older comments, removed

File: acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature [hunk 4]
```diff
@@ -98,16 +98,16 @@ Feature: mod-consortia integration tests
...
     * call read('order-utils/orders.feature')
 
-  Scenario: Create and open order
+  Scenario: Open order with locations from different tenants
     Given call read('features/open-order-with-locations-from-different-tenants.feature')
 
   Scenario: Reopen order and change instance connection orderLine
     Given call read('features/reopen-and-change-instance-connection-order-with-locations-from-different-tenants.feature')
 
-  Scenario: Test cross-tenant inventory objects creation when working with pieces
```
Comment (azizbekxm) [inline acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature:107]: The reason changed title, is to be more clear in cucumber report which scenario belong to which feature

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**